common/djangoapps/philu_commands/management/commands/autoscore_ondemand_course_ora.py

```diff
@@ -18,7 +18,7 @@ class Command(BaseCommand):
     help = """
         Auto Score ORA assesment for On Demand Course if learner has completed all the necessary steps i-e
         submitted their responses, assess required peers responses and finally own assessment if required, if that
-        learner ends up in a waiting queue after 2 weeks of response submission this command will auto-score the
+        learner ends up in a waiting queue after 3 days of response submission this command will auto-score the
```

Comment:
As we are getting it from site configurations and it won't be 3 always. Please update accordingly